packages/react/src/greyVest/utils.js

```diff
@@ -6,36 +6,47 @@ export let openBinding = (...lens) => ({
   onClose: F.off(...lens),
 })
 
+let maxWordsPerTag = 100
+let maxCharsPerTagWord = 100
+
+// Strip these characters when splitting a tag into words. We do this to display
+// tags that are closer to what we send to elastic in a `query_string` query.
+//
+// See https://github.com/smartprocure/contexture-elasticsearch/pull/170
+//
+// If in doubt, make a request to the `/{index}/analyze` elasticsearch endpoint
+// to see exactly which characters get stripped out of text.
+let wordRegex = /[^|><!(){}[\]^"~*?\\;,$']+/g
```

Comment:
Let's consolidate this list with the list here: https://github.com/smartprocure/contexture/pull/205/files#diff-69f43c306be976d94d4493c9ca003a660ca8f6084b779bf99ae2288ca58d4519R38

We can pull this out into a common package that's shared between the front end and back end.

Reply:
Created the common package and moved the regex there 👍